ts/features/payments/home/components/PaymentsHomeTransactionsList.tsx

```diff
@@ -45,21 +55,41 @@ const PaymentsHomeTransactionsList = ({ enforcedLoadingState }: Props) => {
     }
   });
 
-  const handleNavigateToTransactionDetails = (transaction: NoticeListItem) => {
-    if (transaction.eventId === undefined) {
-      return;
+  React.useEffect(() => {
+    if (
+      pot.isSome(latestTransactionsPot) &&
+      !pot.isLoading(latestTransactionsPot)
+    ) {
+      dispatch(
+        clearPaymentsBackoffRetry(PAYMENTS_HOME_TRANSACTIONS_LIST_BACKOFF)
+      );
     }
-    navigation.navigate(
-      PaymentsTransactionBizEventsRoutes.PAYMENT_TRANSACTION_BIZ_EVENTS_NAVIGATOR,
-      {
-        screen:
-          PaymentsTransactionBizEventsRoutes.PAYMENT_TRANSACTION_BIZ_EVENTS_DETAILS,
-        params: {
-          transactionId: transaction.eventId,
-          isPayer: transaction.isPayer
-        }
+  }, [dispatch, latestTransactionsPot]);
+
+  const handleNavigateToTransactionDetails = React.useCallback(
```

Comment:
Since we only need two properties from the object, I suggest the following changes:

```ts
const handleNavigateToTransactionDetails = React.useCallback(
    ({ eventId, isPayer }: NoticeListItem) => {
      if (eventId === undefined) {
        return;
      }
      navigation.navigate(
        PaymentsTransactionBizEventsRoutes.PAYMENT_TRANSACTION_BIZ_EVENTS_NAVIGATOR,
        {
          screen:
            PaymentsTransactionBizEventsRoutes.PAYMENT_TRANSACTION_BIZ_EVENTS_DETAILS,
          params: {
            transactionId: eventId,
            isPayer
          }
        }
      );
    },
    [navigation]
  );
```

Reply:
Good catch! 0846111f80f6e42c659a33d49e2aac30a553ff28